CHANGELOG.md

```diff
@@ -13,6 +13,79 @@
   See [Solutions documentation](https://metal-k8s.readthedocs.io/en/development-2.5/developer/solutions/archive.html#product-information)
   for details about the new format.
 
+### Bug fixes
+- [#2434](https://github.com/scality/metalk8s/issues/2434) - Add Wait for a
+single Salt Master container during Bootstrap
+(PR [#2435](https://github.com/scality/metalk8s/pull/2435))
+
+- [#2526](https://github.com/scality/metalk8s/issues/2526) -Add 'groups'
+scope when requesting an id_token from Dex
+(PR [#2529](https://github.com/scality/metalk8s/pull/2529))
+
+- [#2443](https://github.com/scality/metalk8s/issues/2443) - Improve error
+handling for salt jobs in the UI
+(PR [#2475](https://github.com/scality/metalk8s/pull/2475))
+
+- [#2495](https://github.com/scality/metalk8s/issues/2495) - Fix monitoring
+page to display all alerts
+(PR [#2503](https://github.com/scality/metalk8s/pull/2503))
+
+### Enhancements
+- [#1461](https://github.com/scality/metalk8s/issues/1461) - Improve JSON
+parsing in storage-operator
+(PR [#2577](https://github.com/scality/metalk8s/pull/2577))
+
+- [#2422](https://github.com/scality/metalk8s/issues/2422) - Replace
+config.yaml + product.txt with a single Solutions manifest.yaml
+(PR [#2565](https://github.com/scality/metalk8s/pull/2565))
```

Comment:
I think this one should be removed since it's already in breaking changes section